src/components/Contact/contactitem.tsx

```diff
@@ -22,12 +23,11 @@ const ContactItem = ({ contact, image, firestoreIndex }: ContactItemProps) => {
         {/* USER PROFILE IMAGE */}
         <span className='flex items-center space-x-4'>
           <Avatar image={image} height={40} width={40} iconClass='h-10 w-10' />
-          <p className='font-medium text-gray-700'>
-            {contact.clientName}
-            {/* {contacts[firestoreIndex].clientName} */}
-          </p>
+          <p className='font-medium text-gray-700'>{contact.clientName}</p>
         </span>
-        <p className='text-gray-500'>{truncateText(contact.pets, 16)}</p>
+        {contact.pets && (
+          <p className='text-gray-500'>{truncateText(contact.pets, 16)}</p>
```

Comment:
we can also truncate text with css
https://css-tricks.com/snippets/css/truncate-string-with-ellipsis/